parameter_files/fates_params_default.cdl

```diff
@@ -1597,6 +1624,10 @@ data:
 
  fates_frag_cwd_frac = 0.045, 0.075, 0.21, 0.67 ;
 
+ fates_landuse_crop_lu_pft_vector = -999, -999, -999, -999, 11 ;
+
+ fates_max_nocomp_pfts_by_landuse = 4, 4, 2, 2, 1 ;
```

Comment:
noting that these numbers need to change so that they are each <= `fates_maxpatches_by_landuse`